tests/unit/test_app_runners.py

```diff
@@ -0,0 +1,34 @@
+import time
+import sys
+import requests
+import pytest
+
+import dash_html_components as html
+import dash
+
+
+def test_threaded_server_smoke(thread_server):
+    app = dash.Dash(__name__)
+
+    app.layout = html.Div(
+        [
+            html.Button("click me", id="clicker"),
+            html.Div(id="output", children="hello thread"),
+        ]
+    )
+    thread_server(app, debug=True, use_reloader=False, use_debugger=True)
+    time.sleep(0.2)
```

Comment:
Testing the tests 🔁
Normally users won't need to use explicit sleeps like this, right? They'll just do `browser.wait_for_page` or something?

Reply:
yes, this is like a smoke test for server fixtures only, and most of our cases will have that helper API from browser

Reply:
once we stabilize the api, we should further enhance this case with a more comprehensive app layout, so it can cover most of the common usage and alert the bugs at an earlier stage